config/karma.config.js

```diff
@@ -1,39 +1,41 @@
 const webpack = require('webpack');
 const commonConfig = require('./common.webpack.config.js');
 
-module.exports = function(config) {
-    config.set({
-        browsers: ['Firefox'],
-        frameworks: ['mocha'],
-        files: [
-            '../src/**/*.spec.js'
-        ],
+module.exports = function (config) {
+  config.set({
+    browsers: ['PhantomJS'],
```

Comment:
We might leave Firefox as the second browser so we can use it with the `--browser` flag?